my-code/src/styles/_colors.scss

```diff
@@ -0,0 +1,9 @@
+$yellow: #FF9F1C;
```

Comment:
Naming colors variables is always a nightmare, so let me suggest a new pattern we've been working internally:
```scss
// Grayscale
$g100: #eee;
$g200: #222;
$g250: #444;
$g300: #999;

// Variation of branch color
$b100: #353f4c;
$b200: #1b2329;

$error: red;
$sucess: green;
```

Using this approach the variable name becomes more predictable, and the name doesn't carry about its value, for example, what about if the `$red` is not red anymore?

Reply:
Looking at the invision page, colors had names, just thought it was something to follow.